content/en/docs/started/workstation/getting-started-multipass.md

```diff
@@ -10,7 +10,7 @@ weight = 60
 
 ## Introduction
 
-This document outlines the steps that you can take to get your local installation of Kubeflow running on top of [Microk8s](https://microk8s.io/), a small enterprise Kubernetes cluster. Microk8s is now available on Windows, macOS and any Linux distribution that supports `snaps`. You can download it on the [Microk8s](https://microk8s.io/) website. 
+This guide will show you how to deploy Kubeflow using [Microk8s](https://microk8s.io/) - a small enterprise Kubernetes cluster. Microk8s is now available on Windows, macOS and any Linux distribution that supports `snaps`. You can download it on the [Microk8s](https://microk8s.io/) website. 
```

Comment:
After some studying, I realized that using a present tense is the commonly used practice.

```diff
- This guide will show you how to to deploy Kubeflow using [Microk8s]
+ This guide describes how to deploy Kubeflow using [Microk8s]
```

What do you think?

Reply:
LGTM

Reply:
Thank you!